src/components/dialog/dialog.js

```diff
@@ -1,107 +1,293 @@
+import {keyCodes, defaultClassNames, NATIVELY_FOCUSABLE_ELEMENTS} from '../../constants';
+
+import createEl from '../../utils/createEl';
+import defer from '../../utils/defer';
+import qa from '../../utils/qa';
+
+
 /**
  * @function UIDialog
  * @version 0.0.2
+ * @desc Main UIDialog function. Creates instances of the dialog based on
+ * parameters passed in.
+ * @param {object} settings
  */
-
-import {keyCodes, defaultClassNames} from '../../constants';
-
-
 const UIDialog = ({
         dialog = '.js-dialog',
         openBtn = '.js-dialog-btn',
         closeBtn = '.js-dialog-close-btn',
```

Comment:
Should the button classes be:

`.js-dialog-btn-open` (or `trigger`)
`.js-dialog-btn-close`

But more about the open button being more descriptive.

?